### ISSUE TITLE
YelpCamp/routes/comments.js Potentially dangerous code

### ISSUE THREAD

**Issue by michaelKurowski** [Jordan141/YelpCamp#11]
https://github.com/Jordan141/YelpCamp/blob/d997d2227bfd59484fc9c1f8aa8b98aafa3ccc5d/routes/comments.js#L26

What if there is no `comment` property sitting on `req.body`?

**Comment by Jordan141** [comment]
I don't believe the user can access this without returning a comment, I'll check this with tests.

**Comment by michaelKurowski** [comment]
@Jordan141 What if someone will send custom http packet?

**Comment by Jordan141** [comment]
I see the issue now, I forgot to add the login check middleware.

**Comment by michaelKurowski** [comment]
People might still send custom packets with login cache from browser (i.e. postman might use Chrome cookies and attach it to the requests), so that won't prevent anything.

**Comment by Jordan141** [comment]
What do you suggest?

**Comment by michaelKurowski** [comment]
Response something in case if there's an error.
